CONTRIBUTING.md

```diff
@@ -8,24 +8,29 @@ If you can think of something nice to add or want to change / extend some of the
 
 - NodeJS v20
 - npm 11
+- Recommended: Unix-based OS (or WSL on Windows)
 
 ## Setup
 
-To setup a local environment you need to do the following steps:
+To setup a local environment, perform the following steps:
 
 - Fork the repository to your GitHub account, so you can push your changes
 - Clone the repository
 - Install dependencies: `npm install`
 - Run the build `npm run build` (only required once to copy assets)
-- Start the Live Server: `npm run start` or `npm run start:big`
-- If you want to package OctoDash, so you can install it on your Raspberry Pi: `npm run pack`
 
-## Languages
+## Testing
 
-To launch the app in development mode in a specific locale instead of the source english locale, use
+To test locally in a dev environment, simply run `npm start`. The app will automatically reload when changes are made.
+
+If you want to package OctoDash so you can install it on your target device, run `npm run pack`. This will generate `.deb` files in the `package/` folder, which you can upload to your target device for installation and testing in a production environment.
+
+## Localization
+
+To launch the app in development mode in a specific locale instead of the source English locale, use:
 
 ```
-npm run start --serve="--configuration=<lang>"
+npm start --serve="--configuration=<lang>"
```

Comment:
This actually doesn't work anymore as I found out yesterday. We need to put those configurations back in `angular.json`

Reply:
Certainly!  I'm thinking that fixing locale loading should be done in another PR, WDYT?